File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/businessmanagement/cs/lockinvoicepricing.cs
// <snippetlockinvoicepricing>


 // Relock the invoice pricing
 LockInvoicePricingRequest lockInvoiceRequest =
     new LockInvoicePricingRequest()
     {
         InvoiceId = _invoiceId
     };
 _serviceProxy.Execute(lockInvoiceRequest);

 Console.WriteLine("Invoice pricing relocked.");

// </snippetlockinvoicepricing>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/businessmanagementvb/vb/lockinvoicepricing.vb
' <snippetlockinvoicepricing>


 ' Lock the invoice pricing
 Dim lockInvoiceRequest As New LockInvoicePricingRequest() With {.InvoiceId = _invoiceId}
 _serviceProxy.Execute(lockInvoiceRequest)

 Console.WriteLine("Invoice pricing locked.")

 ' Update the price list
 priceListItem.Amount = New Money(40D)

 updatePriceListItem = New UpdateRequest() With {.Target = priceListItem}
 _serviceProxy.Execute(updatePriceListItem)

 Console.WriteLine("Price list updated.")

' </snippetlockinvoicepricing>